templates/agenda/index.html.twig

```diff
@@ -76,17 +78,19 @@
                             <div class="btn-group" role="group" aria-label="{% trans %}List functions{% endtrans %}">
                                 <!-- Show button -->
                                 <a href="{{ url('agenda_show', {id: agenda.id}) }}" role="button" class="btn btn-primary" data-toggle="tooltip" data-placement="auto" title="{% trans %}Show agenda{% endtrans %}"><i class="fas fa-eye"></i><span class="sr-only">{% trans %}Show agenda{% endtrans %}</span></a>
-                                <!-- Delete button -->
-                                {% if agenda.isFinished() %}
-                                    <button class="btn btn-danger" disabled data-toggle="tooltip" data-placement="auto" title="{% trans %}Cannot delete finished agenda{% endtrans %}"><i class="fas fa-trash-alt"></i></button>
-                                {% elseif agenda.agendaItems|length > 0 %}
-                                    <button class="btn btn-danger" disabled data-toggle="tooltip" data-placement="auto" title="{% trans %}Cannot delete agenda containing agenda items{% endtrans %}"><i class="fas fa-trash-alt"></i></button>
-                                {% else %}
-                                <form id="myform" class="btn-group" method="post" action="{{ path('agenda_delete', {'id': agenda.id}) }}" onsubmit="return confirm('{% trans %}Are you sure you want to delete this agenda?{% endtrans %}');">
-                                    <input type="hidden" name="_method" value="DELETE">
-                                    <input type="hidden" name="_token" value="{{ csrf_token('delete' ~ agenda.id) }}">
-                                    <button class="btn btn-danger" data-toggle="tooltip" data-placement="auto" title="{% trans %}Delete agenda{% endtrans %}"><i class="fas fa-trash-alt"></i></button>
-                                </form>
+                                {% if not is_granted('ROLE_BOARD_MEMBER') %}
```

Comment:
Use `{% if is_granted('ROLE_CASEWORKER') %}`

Reply:
Fixed in 4bc1a7723936a2511ae7d457d4dfec56d37c7bf1.